src/main/java/it/smartcommunitylab/aac/config/SecurityConfig.java

```diff
@@ -178,7 +179,7 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
             //                .and()
             .csrf()
             //                .disable()
-            .ignoringAntMatchers("/logout", "/console/**", "/account/**")
+            .ignoringAntMatchers("/logout", "/console/**", "/account/**", EndSessionEndpoint.END_SESSION_URL)
```

Comment:
remove endsession from this filter and move to OAuth2UserSecurityConfig, both for csrf whitelist and permitAll